internals/testing/test-bundler.js

```diff
@@ -9,7 +9,7 @@ import 'jest-localstorage-mock';
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
-global.fetch = require('jest-fetch-mock');
+require('jest-fetch-mock').enableMocks();
```

Comment:
Could we use an import instead of a require?

Reply:
Yeah, could definitely do. I'll change it.